app/views/crime_applications/index.html.erb

```diff
@@ -44,6 +44,7 @@
               <td class="govuk-table__cell">
                 <%= button_to confirm_destroy_crime_application_path(app), method: :get,
                   class: 'govuk-button govuk-button--secondary govuk-!-margin-bottom-0',
+                  'aria-label': "Delete #{app.applicant_name}'s application",
```

Comment:
Might be good to use i18n for this string. Same as we do with the button label itself.
We can call the it something like `delete_button_a11y` to explicitly convey the purpose of this locale.